src/main/java/io/vertx/core/impl/launcher/commands/RunCommand.java

```diff
@@ -274,6 +268,26 @@ public void run() {
     }
   }
 
+  protected JsonObject normalizeConfiguration() {
+    JsonObject conf = getConfiguration();
+    if (conf == null) {
+      conf = new JsonObject();
+    }
+    afterConfigParsed(conf);
+    return conf;
+  }
+
+  protected Vertx doRun() {
+    super.run(this::afterStoppingVertx);
+    return vertx();
+  }
+
+  protected DeploymentOptions normalizeDeploymentOptions(JsonObject conf) {
```

Comment:
rather rename to `buildDeploymentOptions` there is no normalize operation done here

Reply:
Is `configureFromSystemProperties` for DeployOptions considered as `normalize` operation?
If not, I should also rename this method also to `buildVertxOptions` in `BareCommand` to consistence

https://github.com/eclipse-vertx/vert.x/blob/3ee709510091c49d04c11be074b736c199e7ca1f/src/main/java/io/vertx/core/impl/launcher/commands/BareCommand.java#L241-L242